packages/kit/test/build-errors/apps/prerender-impossible-content-negotiation/package.json

```
@@ -0,0 +1,20 @@
+{
+	"name": "prerender-impossible-content-negotiation",
+	"private": true,
+	"version": "0.0.1",
+	"scripts": {
+		"dev": "vite dev",
+		"build": "vite build",
+		"preview": "vite preview",
+		"check": "svelte-kit sync && tsc && svelte-check"
+	},
+	"devDependencies": {
+		"@sveltejs/adapter-auto": "workspace:^",
+		"@sveltejs/kit": "workspace:^",
+		"svelte": "^3.56.0",
+		"svelte-check": "^3.0.2",
+		"typescript": "^4.9.4",
+		"vite": "^4.3.6"
```

we should keep these in sync with the other test projects
```suggestion
		"@sveltejs/adapter-auto": "workspace:^",
		"@sveltejs/kit": "workspace:^",
		"svelte": "^4.0.5",
		"svelte-check": "^3.4.4",
		"typescript": "^4.9.4",
		"vite": "^4.4.9"
```